README.md

```diff
@@ -16,6 +16,21 @@ See https://docs.corda.net/getting-set-up.html.
 
 # Usage
 
+## Running tests inside IntelliJ
+	
+We recommend editing your IntelliJ preferences so that you use the gradle runner - this means that the quasar utils 
+plugin will make sure that these flags are set for you.
```

Comment:
What are "these flags" you're referring to? Confused me a little.

Reply:
Updated - is the new version ok?